create_builder_test.go

```diff
@@ -563,9 +568,9 @@ func testCreateBuilder(t *testing.T, when spec.G, it spec.S) {
 				bldr := successfullyCreateBuilder()
 
 				h.AssertEq(t, bldr.LifecycleDescriptor().Info.Version.String(), "0.0.0")
-				//nolint:staticcheck
+				// nolint:staticcheck
```

Comment:
Guessing this was from some auto-formatting. Wanted to call it out in case the linter will expect a certain format.

Reply:
Fixed